pooltool/physics/resolve/ball_cushion/stronge_compliant/model.py

```diff
@@ -0,0 +1,131 @@
+import logging
+
+import attrs
+import numpy as np
+
+import pooltool.constants as const
+import pooltool.ptmath as ptmath
+from pooltool.objects.ball.datatypes import Ball, BallState
+from pooltool.objects.table.components import (
+    CircularCushionSegment,
+    Cushion,
+    LinearCushionSegment,
+)
+from pooltool.physics.resolve.ball_cushion.core import (
+    CoreBallCCushionCollision,
+    CoreBallLCushionCollision,
+)
+from pooltool.physics.resolve.models import BallCCushionModel, BallLCushionModel
+from pooltool.physics.resolve.stronge_compliant import (
+    resolve_collinear_compliant_frictional_inelastic_collision,
+)
+
+logger = logging.getLogger(__name__)
+ch = logging.StreamHandler()
+ch.setLevel(logging.ERROR)
+logger.addHandler(ch)
+logger.setLevel(logging.ERROR)
```

Comment:
I like that there's logging, and I think we need more of this in pooltool, but I don't like that this adds a StreamHandler at module import time. It doesn't seem like good practice for a library to do, since it forces logging configuration on users. I think we should only create loggers, not configure them.

```suggestion
logger = logging.getLogger(__name__)
```

This gives users full control over logging. They can enable debug logs with:

```
import logging
logging.basicConfig(level=logging.DEBUG)
```

Or keep the default behavior of no output if they don't configure logging.